contracts/dao/voting/investors-vesting-vault/README.md

```diff
@@ -0,0 +1,3 @@
+### Neutron Vesting Vault
+
+This vault will allow its users to query voting power represented by NTRN tokens in the vesting contracts. Just as with normal DAO DAO voting modules, for each specific proposal, you can only use the voting power that was available to you at the time of proposal submission. No additional restrictions are imposed on the vault funds.
```

Comment:
Please add the following:

You can not directly add money to this vault or get it out of this vault. This is a proxy-vault that only queries information from the investors vesting contract.